network-api/networkapi/templates/pages/base.html

```diff
@@ -1,21 +1,40 @@
+{% load wagtailmetadata_tags i18n localization settings_value wagtailuserbar %}
+
+{% get_current_language as lang_code %}
+
 <!DOCTYPE html>
-<html>{% load settings_value wagtailuserbar %}
+<html lang="{{ lang_code }}">
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">{% block social_metadata %}{% endblock %}
+    <title>{% block pageTitle %}Mozilla Foundation - {% if page.specifics.seo_title %}{{ page.specifics.seo_title }}{% else %}{{ page.title }}{% endif %}{% endblock %}</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="csrf-token" content="{{ csrf_token }}">
+
+    {% block social_metadata %}
+      <meta property='og:type' content='website'>
+      <meta property='og:locale' content='{{ lang_code|to_opengraph_locale }}'>
```

Comment:
What's `to_opengraph_locale`? _tried looking it up and got sent to my gitmemory profile...didn't even know that existed_ 😮

Reply:
interesting. It was _supposed_ to be a tag, loaded from localization.py, but that files seems to have disappeared in the PR? O_o

Reply:
that file should now be restored ... so weird